rust/datafusion/src/logical_plan/expr.rs

```diff
@@ -303,6 +311,7 @@ impl Expr {
                 }
             }
             Expr::Cast { expr, .. } => expr.nullable(input_schema),
+            Expr::TryCast { expr, .. } => expr.nullable(input_schema),
```

Comment:
It seems like given the semantics of `TryCast` that this should always return `true`